.changeset/big-glasses-jam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/cypher-builder": patch
```

Comment:
I would say that is a minor change!

Reply:
But when a product is in `0.*.*`, doesn't a patch essentially mean minor, and a minor essentially mean major?

Reply:
Absolutely, my bad!